### PR TITLE
Add update task, bump dependencies

### PR DESCRIPTION
**Pull request by LeoDog896**
This adds a new `update` task to automatically bump all dependencies in the project.

### CODE REVIEW

File: command/upgrade/provider/deno_land.ts [hunk 1]
```diff
@@ -6,8 +6,8 @@ export interface DenoLandProviderOptions {
 
 export class DenoLandProvider extends Provider {
   name = "deno.land";
-  private readonly repositoryUrl = "https://deno.land/x/";
-  private readonly registryUrl = "https://deno.land/x/";
+  private readonly repositoryUrl = "https://deno.land/x";
+  private readonly registryUrl = "https://deno.land/x";
```
Comment (c4spar) [inline command/upgrade/provider/deno_land.ts:10]: Was this change intended or is this a bug in deno-outdated? Because the trailing slash is required here.

**Review by c4spar — CHANGES_REQUESTED** [review]
Hi @LeoDog896, sry for the late response and thx for updating the dependencies!
I left one comment. Looks like deno-outdated has a bug.

FYI: Are you aware that there is already a tool to update outdated dependencies? [deno-udd](https://github.com/hayd/deno-udd)

File: command/upgrade/provider/deno_land.ts (hunk 1 above)
Comment (LeoDog896) [reply command/upgrade/provider/deno_land.ts:10]: Got it, I'll use udd instead for this PR.

Comment (LeoDog896) [reply command/upgrade/provider/deno_land.ts]: I ended up still using deno-outdated until my discussion in deno is resolved: https://github.com/denoland/deno/discussions/15625 as globs can't be passed to check all the *.ts files in this repository. When this is resolved, I'll make another PR to switch to udd.

Comment (c4spar) [reply command/upgrade/provider/deno_land.ts]: Thx for fixing 👍

**Review by c4spar — APPROVED** [review]
LGTM! Thx @LeoDog896
